### DISCUSSION TITLE
colors are not saved

### DISCUSSION THREAD

**Discussion by Galaeasy013274** [mriscoc/Ender3V2S1#673] [General]
Hello, in the Ender3V2S1-20221222 firmware when I change the colors from the display and then turn off the printer when I turn it on again the set colors are not saved

**Comment by mriscoc** [comment]
Use save settings to write into the EEPROM all your changes

**Comment by Galaeasy013274** [comment]
I did it but it doesn't work

**Reply by mriscoc** [reply]
Let me check

**Reply by mriscoc** [reply]
Confirmed, Thanks. But the colors are indeed saved but not applied at restart, please go to Control menu and do a click in load settings, it will be fixed in the next release.
